data/handler/handler.go

```diff
@@ -0,0 +1,106 @@
+package handler
+
+import (
+	"encoding/csv"
+	"io"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/wdonet/golang-bootcamp-2020/domain/model"
+	"github.com/wdonet/golang-bootcamp-2020/utilities"
+)
+
+const filename string = "./data/todos.csv"
+
+// GetTodosFromFile get data from files as Todos
+func GetTodosFromFile() []*model.Todo {
+	csvfile, err := os.Open(filename)
+	if err != nil {
+		log.Println("Unable to open for read CSV file!", err)
+		return nil
+	}
+	r := csv.NewReader(csvfile)
+	r.TrimLeadingSpace = true
+
+	var numOfRecords int = 0
```

Comment:
```suggestion
	numOfRecords := 0
```

Just a shorthand expression.